### ISSUE TITLE
Problem with restoring cropPosition when bitmap is scrolled to edge

### ISSUE THREAD

**Issue by onlyway** [naver/android-imagecropview#17]
When I want to restore bitmap somewhere centered everything is working fine, but when I'm trying to scroll max to top or bottom, after restoration I'm not exactrly at the same position. I have margin from top or from bottom. Is there any workaround for this?

**Comment by onlyway** [comment]
In sample app I see the same bug with edges

**Comment by helloyako** [comment]
@onlyway 
Yes, that is bug.. I'm so sorry. 😭

**Comment by onlyway** [comment]
Do you have any walk around for this?

**Comment by helloyako** [comment]
Not yet

**Comment by onlyway** [comment]
Ok thx, i've tried to solve this couple of Tomek but no success :)

**Comment by helloyako** [comment]
@onlyway 
How are you? :)
I fixed this issue.

I will include this spec in v1.2.0. but it's not released.
You can use 1.2.0-snapshot or develop branch source code.

### snapshot version from maven
```
repositories {
    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
}
dependencies {
   compile 'com.naver.android.helloyako:imagecropview:1.2.0-SNAPSHOT'
}
```

check please

**Comment by helloyako** [comment]
resolved v1.2.0

**Comment by onlyway** [comment]
Thx ;) I'll check it later
